eqdsk/cocos.py

```diff
@@ -352,7 +352,7 @@ def identify_cocos(
         raise ValueError(
             "The sign of qpsi is not consistent across the flux surfaces.",
         )
-    sign_q = Sign(sign_q.max())
+    sign_q = Sign(int(sign_q.max()) or 1)
```

Comment:
0 is not a valid Sign neither is 1.0 therefore int to get an integer if integer is 0 `0 or 1 == 1`